src/components/SRNEntry.vue

```diff
@@ -235,9 +237,8 @@ export default {
         this.userIDList[index]["userID"] = event.target.value.toString();
       }
     },
+    //incorrect entry the first time
```

Comment:
I don't think this comment is valid here right? This method handles incorrect entries and that's it. This doesn't know or care if the entry is being made the first time or nth time.
We care about that in the parent methods where this method is being called

Instead of this comment, I would suggest writing how ANY incorrect entry is handled
Also I think we should get into the habit of writing multi line comments!
For example, the below comment tells us what the function does, and if there are any parameters that need to be passed to the function.
```js
/**
 * Comment goes here - what does the function do?
 * @param {Number} - param1Name
 * @param {String} - param2Name 
 */
```
These comments should be put on top of a function. No need of multiline comments inside a function

Also, no need to change all of them now, but slowly slowly we should convert all our function docstrings into multi line docstrings